tasks/sep/020-holocene-system-config-upgrade-multi-chain/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+import {stdJson} from "forge-std/stdJson.sol";
+
+/// @title ISemver
+/// @notice ISemver is a simple contract for ensuring that contracts are
+///         versioned using semantic versioning.
+interface ISemver {
+    /// @notice Getter for the semantic version of the contract. This is not
+    ///         meant to be used onchain but instead meant to be used by offchain
+    ///         tooling.
+    /// @return Semver contract version as a string.
+    function version() external view returns (string memory);
+}
+
+contract NestedSignFromJson is OriginalNestedSignFromJson {
+    string[4] l2ChainIds = [
+        "11155420", // op
+        "1740", // metal
+        "919", // mode
+        "999999999" // zora
+    ];
+
+    address livenessGuard = 0xc26977310bC89DAee5823C2e2a73195E85382cC7;
+
+    address newSystemConfigImplAddress = 0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD;
+
+    // Safe contract for this task.
+    GnosisSafe securityCouncilSafe = GnosisSafe(payable(vm.envAddress("COUNCIL_SAFE")));
+    GnosisSafe fndSafe = GnosisSafe(payable(vm.envAddress("FOUNDATION_SAFE")));
+    GnosisSafe ownerSafe = GnosisSafe(payable(vm.envAddress("OWNER_SAFE")));
```

Comment:
In the setup you can run some precheck assertions to make sure these env vars were set correctly, such as https://github.com/ethereum-optimism/superchain-ops/blob/c9a5f9b2ff8cfabd143ef5bc1eabdb0349632b27/tasks/eth/010-1-guardian-upgrade/NestedSignFromJson.s.sol#L85

Reply:
[4413fb5](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/4413fb5f4ebec49179cc9f6c55410ece310cc85e)